features/card_payments_common.feature

```diff
@@ -0,0 +1,69 @@
+Feature: Card Payments
+
+  As a user
+  I want to use card payments method
+  In order to check full payment functionality
+
+  Background:
+    Given JavaScript configuration is set for scenario based on scenario's @config tag
+    And User opens page with payment form
+
+  @base_config @extended_tests_part_1
+  Scenario Outline: Payment form accessibility after payment process
+    When User fills payment form with credit card number "4000000000001000", expiration date "12/22" and cvv "123"
+    And THREEDQUERY mock response is set to "NOT_ENROLLED_N"
+    And User clicks Pay button - AUTH response is set to "<action_code>"
+    Then User will see that Submit button is "<form_status>" after payment
+    And User will see that all input fields are "<form_status>"
+
+    @smoke_test
+    Examples:
+      | action_code | form_status |
+      | OK          | disabled    |
+
+    Examples:
+      | action_code | form_status |
+      | DECLINE     | enabled     |
+
+  @config_default
+  Scenario: Checking that 'submitOnSuccess' is enabled by default
```

Comment:
This scenario and rest with submitOnSuccess / OnError can be move to separete feature